classes.fields.php

```diff
@@ -1914,6 +1915,9 @@ public function display() {
 
 		if ( ! $this->args['repeatable'] && empty( $values ) ) {
 			$values = array( null );
+		} else {
+			$values = $this->get_values(); // Make PHP5.4 >= happy.
+			$values = ( empty( $values ) ) ? array( '' ) : $values;
```

Comment:
Does this mean that the `if ( $values )` below is now redundant? `$values` should always have >= 1 element, so `$values` will be truthy I think